[10:25:50] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 404 41 - 10.002 ms[39m
[35m[39m
[10:27:10] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 404 41 - 13.204 ms[39m
[35m[39m
[10:27:14] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 404 41 - 2.159 ms[39m
[35m[39m
[10:27:51] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 400 56 - 11.180 ms[39m
[35m[39m
[10:28:24] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 266.095 ms[39m
[35m[39m
[10:29:31] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 41.334 ms[39m
[35m[39m
[10:29:36] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 7.523 ms[39m
[35m[39m
[10:31:08] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 39.271 ms[39m
[35m[39m
[10:32:31] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 24.417 ms[39m
[35m[39m
[10:32:36] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 500 92 - 4.681 ms[39m
[35m[39m
[10:38:19] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 200 262 - 43.371 ms[39m
[35m[39m
[10:38:57] [DEV] [[35mhttp[39m]: [35m::1 POST /api/v1/admin/login 200 262 - 6.907 ms[39m
[35m[39m
[20:44:14] [DEV] [[31merror[39m]: [31munhandledRejection: column "createdAt" of relation "Users" contains null values[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async User.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[20:46:30] [DEV] [[31merror[39m]: [31munhandledRejection: column "createdAt" of relation "Users" contains null values[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async User.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[20:53:01] [DEV] [[31merror[39m]: [31munhandledRejection: column "order" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[20:53:01] [DEV] [[31merror[39m]: [31munhandledRejection: column "order" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:967:13)[39m
[20:54:41] [DEV] [[31merror[39m]: [31munhandledRejection: column "url" of relation "PostImages" already exists[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostgresQueryInterface.addColumn (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\abstract\query-interface.js:184:12)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:958:11)[39m
[20:54:41] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[20:58:14] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[20:58:14] [DEV] [[31merror[39m]: [31munhandledRejection: column "userId" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:967:13)[39m
[21:00:23] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:02:08] [DEV] [[31merror[39m]: [31munhandledRejection: column "url" of relation "PostImages" already exists[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostgresQueryInterface.addColumn (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\abstract\query-interface.js:184:12)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:958:11)[39m
[21:02:47] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:03:21] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:04:37] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:09:10] [DEV] [[31merror[39m]: [31munhandledRejection: No description found for "PostImages" table. Check the table name and schema; remember, they _are_ case sensitive.[39m
[31mError: No description found for "PostImages" table. Check the table name and schema; remember, they _are_ case sensitive.[39m
[31m    at PostgresQueryInterface.describeTable (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\abstract\query-interface.js:168:15)[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Promise.all (index 0)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:947:26)[39m
[21:09:10] [DEV] [[31merror[39m]: [31munhandledRejection: No description found for "PostImages" table. Check the table name and schema; remember, they _are_ case sensitive.[39m
[31mError: No description found for "PostImages" table. Check the table name and schema; remember, they _are_ case sensitive.[39m
[31m    at PostgresQueryInterface.describeTable (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\abstract\query-interface.js:168:15)[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Promise.all (index 0)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:947:26)[39m
[21:10:10] [DEV] [[31merror[39m]: [31munhandledRejection: column "order" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:967:13)[39m
[21:12:09] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:12:35] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:13:09] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:14:17] [DEV] [[31merror[39m]: [31munhandledRejection: column "userId" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:16:58] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:16:59] [DEV] [[31merror[39m]: [31munhandledRejection: column "targetId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:18:04] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:18:56] [DEV] [[31merror[39m]: [31munhandledRejection: column "userId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:21:06] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:22:57] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:22:57] [DEV] [[31merror[39m]: [31munhandledRejection: column "targetId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:23:27] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:24:53] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:25:20] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:25:20] [DEV] [[31merror[39m]: [31munhandledRejection: column "userId" of relation "PostImages" does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:25:57] [DEV] [[31merror[39m]: [31munhandledRejection: Unknown constraint error[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async PostImage.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:979:17)[39m
[21:25:57] [DEV] [[31merror[39m]: [31munhandledRejection: column "targetId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:26:44] [DEV] [[31merror[39m]: [31munhandledRejection: column "targetId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
[21:27:51] [DEV] [[31merror[39m]: [31munhandledRejection: column "targetId" referenced in foreign key constraint does not exist[39m
[31mError[39m
[31m    at Query.run (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)[39m
[31m    at C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\sequelize.js:315:28[39m
[31m    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Block.sync (C:\Users\fptshop\Workspace\SNS-Deepfake\server\node_modules\sequelize\lib\model.js:984:11)[39m
